devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1318,6 +1318,9 @@ bool WholeBodyDynamicsDevice::open(os::Searchable& config)
 {
     std::lock_guard<std::mutex> guard(this->deviceMutex);
 
+    double tick = yarp::os::Time::now();
+    yDebug() << "wholeBodyDynamics Statistics: Opening estimator.";
```

Comment:
Isn't it better to call tick after the debug print? Print is time-expensive, maybe?

Reply:
- Print 
- tick
- tock
- Print

Reply:
Done in https://github.com/robotology/whole-body-estimators/pull/106/commits/65fabfd81f7fc25a246d80a61bd3e0dc0ae0df20